tools/setup/emoji/build_emoji

```diff
@@ -423,6 +423,14 @@ def dump_emojis(cache_path: str) -> None:
     EMOJI_DATA_FILE_PATH = os.path.join(NODE_MODULES_PATH, "emoji-datasource-google", "emoji.json")
     with open(EMOJI_DATA_FILE_PATH, "rb") as emoji_data_file:
         emoji_data = orjson.loads(emoji_data_file.read())
+
+    # These are the codes that we'll be able to show Google Modern
+    # emoji for. (For other emoji sets, we fall back to Google Modern.)
+    supported_codes = {get_emoji_code(emoji) for emoji in emoji_data if emoji_is_supported(emoji)}
+    # These are in the emoji dropdown so we should make sure they're supported.
+    for code in EMOJI_NAME_MAPS:
+        assert code in supported_codes
```

Comment:
Can we add a message on this assert specifying what you're supposed to when this happens?

Reply:
Sure, added one.